src/test/java/mobiletestrunner/SignUpTest.java

```diff
@@ -0,0 +1,69 @@
+package mobiletestrunner;
+
+import java.util.HashMap;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import io.appium.java_client.MobileBy;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.touch.offset.PointOption;
+import mobilepageobjects.LandingPage;
+import mobilepageobjects.SignUpFormPage;
+
+public class SignUpTest extends Hooks {
+	
+	//Test case for inputting credentials into sign up form. Does not complete registration to avoid flags
+	@Test(dataProvider = "sign-up-creds")
+	public void signUpTest(
+				String firstName,
+				String lastName,
+				String email,
+				String password,
+				String mobile,
+				String zipCode
+			) throws InterruptedException {
+		Thread.sleep(3000);
+		LandingPage landing = new LandingPage(driver);
+		SignUpFormPage signUp = new SignUpFormPage(driver);
+		
+		String tosAcceptSelector = "new UiSelector().className(\"android.widget.Button\").text(\"ACCEPT\")";
```

Comment:
There is already a method in the LandingPage class called "acceptButton" to accept the prompt when you first open the app.

Reply:
This is prior to you pushing your PR to master so it's not there in this version